docs/Data_Submission_Portal/Users_Guide/Troubleshooting.md

```diff
@@ -0,0 +1,1236 @@
+
+## Transactions Page
+
+After attempting data upload, the Data Submission Portal's [transaction page](https://docs.gdc.cancer.gov/Data_Submission_Portal/Users_Guide/Data_Submission_Process/#transactions) will indicate the state of each transaction as `SUCCEEDED`, `PENDING`, or `FAILED`. 
+
+[![GDC Submission Transaction Page](images/DSP_Transaction_Page.png)](images/DSP_Transaction_Page.png "Click to see the full image.")
+
+### Transaction Details
+
+Clicking on a failed upload will open the [details panel](https://docs.gdc.cancer.gov/Data_Submission_Portal/Users_Guide/Data_Submission_Process/#transactions-details), which includes a Documents section containing an Error Report. 
+
+[![GDC Submission Transaction Details](images/DSP_Transaction_Details.png)](images/DSP_Transaction_Details.png "Click to see the full image.")
+
+## Error Report
+
+The Error Report is generated in tab-delimited format and describes the reason(s) that the transaction was not successful.
+
+### Error Report Table
+
+The Error Report table displays the following information:
+
+|Column|Description|
+| --- | --- |
+| __entityType__ | Entity type that caused the error |
+| __key__ | Property that caused the error|
+| __errorType__ | The type of error |
+| __message__ | Detailed description of the error |
+| __id__ | UUID of the errored entity |
+
+[![GDC Submission Transaction Details](images/DSP_Error_Report.png)](images/DSP_Error_Report.png "Click to see the full image.")
+
+### Data Upload Error Messages
+
+Each error type can have numerous error messages which are detailed in the following sections. If data upload is attempted through the API, the command line output will indicate whether the transaction was successful or failed, and, if applicable, the same error message(s) that would be included in the Data Submission Error Report.
+
+#### ERROR Messages
+
+|Message|Explanation|Solution|
+| --- | --- | --- |
+| __'{Value}' is not one of [{acceptable values}]__ | The value is not accepted by the GDC for the designated property | Ensure the property value is acceptable by reviewing the [GDC Data Dictionary](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/) |
+| __'{Entity}' with {'project_id': '{project_id}', 'submitter_id': '{entity submitter_id}'} already exists in the GDC__ | An entity with that submitter_id has already been uploaded to the designated project | Ensure the submitter_id is unique to the project |
+| __Additional properties are not allowed ('{property}' or '{list of properties}') was/were unexpected__ | The given property or properties are not accepted for the designated entity | Ensure entity accepts the properties by reviewing the [GDC Data Dictionary](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/) |
+| __{value} is less than the minimum of -32872__ | The amount is less than the minimum accepted value | Ensure the value is greater than or equal to the minimum value |
```

Comment:
For this one, recommend that they view the best practices section on Date Obfuscation if applicable:  https://docs.gdc.cancer.gov/Data_Submission_Portal/Users_Guide/Best_Practices/#date-obfuscation